src/templates/FleaMarket/AllProducts.tsx

```diff
@@ -36,25 +36,30 @@ export default function AllProducts() {
       const pageSize = getPageSize(deviceState, DEVICE_PRODUCT_COUNT);
       const order = orderState === "최신순" ? "recent" : "favorite";
 
-      const data = await getProductsDataAsync({
-        order,
-        page: currentPage,
-        pageSize,
-        keyword,
+      const res = await axiosFetch({
```

Comment:
혹시 여기서 res에 대한 타입이 원하는대로 올바르게 추론되나요? :) 
값이 만들어지는 순간 해당 타입을 가질수있게끔 하려면 어떻게 해야할까요?
제네릭을 써주면 사용처에서 res에 대한 타입을 지정해줄수있겠죠?

```suggestion
      const res = await axiosFetch<Response>({
```